### ISSUE TITLE
Turrets laser projectiles sometime won't register the hit onto mobs when theres too many.

### ISSUE THREAD

**Issue by tankyap** [x3rdev/MekanismTurrets#15]
https://github.com/user-attachments/assets/94b5580f-f168-4936-a4ac-7d0fc8177c7e

The turret works normally when it only has 8 speed upgrades, but if you apply more than 8 speed upgrades (can be done by using Mekanism Tweaks addon after tweaking the mod files to work on 1.20.1) or using a tile accelerator like Dark Pedestal w/ Time Watch from ProjectE, some of the projectiles suddenly doesn't work. 

The issue can be ignored for balencing reason but the turret is jst too slow for default configs

**Comment by FuryJack07** [comment]
I think this is just invincibility frames being perfectly timed for speed up 8 turrets. so 9+ upgrades sometimes causes a shot to go through a mob while it's invincible and therefore not get registered

**Comment by x3rdev** [comment]
Yeah this is not going to be fixed, FuryJack07 is correct, the ultimate turret is timed for the default max 8 speed upgrades.
